licenses/license-engine.sh

```diff
@@ -13,6 +13,8 @@ ALLOWED_LICENSES=(
 	'Apache-2.0'
 	'Apache License, Version 2.0'
 	'Apache*'
+	'Artistic-2.0'
+	'BlueOak-1.0.0'
```

Comment:
It seems like this is fine, because it is intended as an easier to read alternative to MIT: https://blueoakcouncil.org/license/1.0.0

And looking at the dependencies, it looks like it would be pretty hard to remove the affected packages.